libcontainer/intelrdt/intelrdt.go

```diff
@@ -229,15 +229,18 @@ func init() {
 		}
 	}
 
-	if flagsSet.MBMTotal || flagsSet.MBMLocal {
+	if flagsSet.MBMTotal || flagsSet.MBMLocal || flagsSet.CMT {
 		if _, err := os.Stat(filepath.Join(intelRdtRoot, "info", "L3_MON")); err == nil {
```

Comment:
To avoid too much nesting, we can return early:
```suggestion
		if _, err := os.Stat(filepath.Join(intelRdtRoot, "info", "L3_MON")); err != nil {
			return
		}
```

Reply:
@kolyshkin Good idea. Thank you.